ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -62,6 +62,20 @@ tests:
           cpu: "2"
           memory: 4Gi
     workflow: ipi-aws
+- as: e2e-console
+  run_if_changed: frontend/
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    test:
+    - as: console-test
+      cli: latest
+      commands: frontend/console-test-frontend.sh
+      from: bin
+      resources:
+        requests:
+          cpu: "2"
+          memory: 4Gi
```

Comment:
@yapei per latest information, 4Gi is expensive. could you please reduce it again? like 600Mi.
I suggest you try it. if the case works, take this value.